projects/1.16/assets/prehistoric-fauna/prehistoricfauna/lang/zh_cn.json

```
@@ -0,0 +1,968 @@
+	"block.prehistoricfauna.protopiceoxylon_log": "原始云杉原木",
+	"block.prehistoricfauna.metasequoia_log": "水杉原木",
+	"block.prehistoricfauna.stripped_metasequoia_log": "去皮水杉原木",
+	"block.prehistoricfauna.metasequoia_wood": "水杉木",
+	"block.prehistoricfauna.stripped_metasequoia_wood": "去皮水杉木",
+	"block.prehistoricfauna.araucaria_log": "南洋杉原木",
+	"block.prehistoricfauna.araucaria_wood": "南洋杉木",
+	"block.prehistoricfauna.stripped_araucaria_log": "去皮南洋杉原木",
+	"block.prehistoricfauna.stripped_araucaria_wood": "去皮南洋杉木",
+	"block.prehistoricfauna.cypress_log": "落羽松原木",
+	"block.prehistoricfauna.stripped_cypress_log": "去皮落羽松原木",
+	"block.prehistoricfauna.cypress_wood": "落羽松木",
+	"block.prehistoricfauna.stripped_cypress_wood": "去皮落羽松木",
+	"block.prehistoricfauna.ginkgo_log": "银杏原木",
+	"block.prehistoricfauna.ginkgo_wood": "银杏木",
+	"block.prehistoricfauna.stripped_ginkgo_log": "去皮银杏原木",
+	"block.prehistoricfauna.stripped_ginkgo_wood": "去皮银杏木",
+	
+	"block.prehistoricfauna.liriodendrites_slab": "鹅掌楸台阶",
+	"block.prehistoricfauna.heidiphyllum_slab": "海蒂菲兰台阶",
+	"block.prehistoricfauna.protojuniperoxylon_slab": "原始刺柏台阶",
+	"block.prehistoricfauna.zamites_slab": "拟查米亚台阶",
+	"block.prehistoricfauna.protopiceoxylon_slab": "原始云杉台阶",
+	"block.prehistoricfauna.araucaria_slab": "南洋杉台阶",
+	"block.prehistoricfauna.metasequoia_slab": "水杉台阶",
+	"block.prehistoricfauna.cypress_slab": "落羽松台阶",
+	"block.prehistoricfauna.ginkgo_slab": "银杏台阶",
+	
+	"block.prehistoricfauna.liriodendrites_stairs": "鹅掌楸楼梯",
+	"block.prehistoricfauna.heidiphyllum_stairs": "海蒂菲兰楼梯",
+	"block.prehistoricfauna.protojuniperoxylon_stairs": "原始刺柏楼梯",
+	"block.prehistoricfauna.zamites_stairs": "拟查米亚楼梯",
+	"block.prehistoricfauna.protopiceoxylon_stairs": "原始云杉楼梯",
+	"block.prehistoricfauna.araucaria_stairs": "南洋杉楼梯",
+	"block.prehistoricfauna.metasequoia_stairs": "水杉楼梯",
+	"block.prehistoricfauna.cypress_stairs": "落羽松楼梯",
+	"block.prehistoricfauna.ginkgo_stairs": "银杏楼梯",
+	
+	"block.prehistoricfauna.liriodendrites_leaves": "鹅掌楸树叶",
+	"block.prehistoricfauna.liriodendrites_leaves_flower": "开花鹅掌楸树叶",
+	"block.prehistoricfauna.heidiphyllum_leaves": "海蒂菲兰树叶",
+	"block.prehistoricfauna.metasequoia_leaves": "水杉树叶",
+	"block.prehistoricfauna.protojuniperoxylon_leaves": "原始刺柏树叶",
+	"block.prehistoricfauna.zamites_leaves": "拟查米亚树锥",
+	"block.prehistoricfauna.protopiceoxylon_leaves": "原始云杉树叶",
+	"block.prehistoricfauna.araucaria_leaves": "南洋杉树叶",
+	"block.prehistoricfauna.cypress_leaves": "落羽松树叶",
+	"block.prehistoricfauna.ginkgo_leaves": "银杏树叶",
+	
+	"block.prehistoricfauna.liriodendrites_pressure_plate": "鹅掌楸压力板",
+	"block.prehistoricfauna.heidiphyllum_pressure_plate": "海蒂菲兰压力板",
+	"block.prehistoricfauna.protojuniperoxylon_pressure_plate": "原始刺柏压力板",
+	"block.prehistoricfauna.zamites_pressure_plate": "拟查米亚压力板",
+	"block.prehistoricfauna.protopiceoxylon_pressure_plate": "原始云杉压力板",
+	"block.prehistoricfauna.araucaria_pressure_plate": "南洋杉压力板",
+	"block.prehistoricfauna.metasequoia_pressure_plate": "水杉压力板",
```

```suggestion
	"block.prehistoricfauna.protopiceoxylon_pressure_plate": "原始云杉木压力板",
	"block.prehistoricfauna.araucaria_pressure_plate": "南洋杉木压力板",
	"block.prehistoricfauna.metasequoia_pressure_plate": "水杉木压力板",
```
同样的，原版有“云杉木压力板”，所以“木”字应该是有必要的